src/state/CallViewModel.ts

```diff
@@ -403,7 +414,7 @@ export class CallViewModel extends ViewModel {
   private readonly remoteParticipantHolds$ = this.scope.behavior<
```

Comment:
Irrelevant for this PR but a general question:
 - do we still need to concept of holds?
    - Tiles are based on the matrix room state memberhsips -> sfu switches should be trivial (we still show the tile in connection state until we find the media on the new sfu)
    - they will be hold on screen anyways even if lk connection is lost due to being based on member events.